packages/core/src/routes/experience/classes/experience-interaction.ts

```diff
@@ -192,7 +181,13 @@ export default class ExperienceInteraction {
    * @throws RequestError with 400 if the interaction event is not `ForgotPassword` and the current interaction event is `ForgotPassword`
    */
   public async setInteractionEvent(interactionEvent: InteractionEvent) {
-    await this.signInExperienceValidator.guardInteractionEvent(interactionEvent);
+    const oneTimeTokenVerificationRecord = this.verificationRecords.get(
+      VerificationType.OneTimeToken
+    );
+    await this.signInExperienceValidator.guardInteractionEvent(
+      interactionEvent,
+      oneTimeTokenVerificationRecord
```

Comment:
Make it a boolean value? Like `isOneTimeTokenVerified`

Reply:
Updated it to `hasVerifiedOneTimeToken?: boolean`